src/components/addQuestion/index.js

```diff
@@ -0,0 +1,69 @@
+import { connect } from 'react-redux';
+import { createQuestion } from '../../store/question.js';
+
+import './index.css';
+
+let AddQuestionForm = ({ onSave, positionClass }) => {
+	const onSubmit = (event) => {
+		event.preventDefault();
+		const {
+			question: { value: question },
+			askee: { value: askee },
+			status: { value: status },
+		} = event.target;
+		onSave({ question, askee, status });

```

Comment:
I would use `createQuestion` instead of `onSave` for the name of this function to make it more clear exactly what is happening here.